cfpb/consumerfinance.gov
# FIG: Improve block naming, conditionally show section IDs

Improve admin UX and visual design for the FIG.

## Changes

- Rename "Fig Sub 3 Section" to "Fig Level 3 Subsection" in the Wagtail admin block chooser
- Rename "Fig Sub Section" to "Fig Subsection" in the Wagtail admin block chooser
- Add help text to section headers to show what header level they display as (h2, h3, or h4) in the Wagtail admin
- When the FIG is displayed, don't show a section ID for level 3 subsection headers. Only show section IDs for sections and subsections.


## How to test this PR

1. Run this branch.
2. Create a Fig Content Page.
3. Use the wagtail admin to add all kinds of blocks (section, subsection, and level 3 subsection) and see that they're all labeled consistently in the block chooser.
4. Preview or publish the page, and see that level 3 subsections don't get a section ID in the header when they're displayed in the body of the page.


## Screenshots

The body of the page when published:
![Screen Shot 2022-06-28 at 10 00 44 AM](https://user-images.githubusercontent.com/4295388/176197969-203eb866-9386-47bf-aa20-c504c61492f3.png)

The admin view:
![Screen Shot 2022-06-28 at 10 03 32 AM](https://user-images.githubusercontent.com/4295388/176198617-e84fa18f-3955-454a-a803-e644286ba87f.png)


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
